ludwig/models/gbm.py

```diff
@@ -12,7 +12,7 @@
 from ludwig.features.base_feature import OutputFeature
 from ludwig.globals import MODEL_WEIGHTS_FILE_NAME
 from ludwig.models.base import BaseModel
-from ludwig.schema.model_config import ModelConfig, OutputFeaturesContainer
+from ludwig.schema.config.model_config import ModelConfig, OutputFeaturesContainer
```

Comment:
No need to do it in this PR, but if you wanted to graduate ModelConfig up to the module level, I would support importing it in `__init__.py` so that we can do:

```
from ludwig.schema import ModelConfig
```

This can be done in a later pass though: if we don't do this now I might when we add API annotations